backend/src/api/webhooks/groupsio.ts

```diff
@@ -0,0 +1,56 @@
+import IntegrationRepository from '../../database/repositories/integrationRepository'
+import SequelizeRepository from '../../database/repositories/sequelizeRepository'
+import IncomingWebhookRepository from '../../database/repositories/incomingWebhookRepository'
+import { WebhookType } from '../../types/webhooks'
+import { getIntegrationStreamWorkerEmitter } from '@/serverless/utils/serviceSQS'
+
+export default async (req, res) => {
```

Comment:
You might want to move this part to the new webhook-api service?

Reply:
Moved :)